src/main/scala/osgifelix/FelixRunner.scala

```diff
@@ -27,6 +27,7 @@ object FelixRunner {
   def startFramework(startConfig: BundleStartConfig, storageDir: File): Felix = {
     val sysO = startConfig.extraSystemPackages.headOption.map(_ => startConfig.extraSystemPackages.mkString(","))
     val configMap: Map[String, String] = Seq(
+      Some(Constants.FRAMEWORK_BOOTDELEGATION -> "sun.misc"),
```

Comment:
Probably it's better to be able to configure boot delegations via build.sbt or to add any properties to framework configuration